src/components/Card/CourseCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react'
+import PlayButton from '../Button/PlayButton'
+import Card from './'
+
+const expansions = [false, 'horizontal', 'vertical']
+
+export const CourseMeta = ({meta}) => {
```

Comment:
@trevordmiller I had mixed feelings about this... I did it this way specifically for the example Cards to be put together from the `cardTypes` map in `./Cards/index.js` but we could refactor Card to specifically accept a `metaComponent` ?